PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+## Pull request title
+
+<!-- Open this PR as draft while it is not ready -->
+<!-- Use keywords like 'close' or 'solves' to link this PR to an issue -->
+
+This PR solves #12345
```

Comment:
Instead of this, we can have a `Related Issues` section:
```markdown
### Related Issues
<!--
Closes #12345
Unblocks #54321
-->
```

Reply:
Related Issues section created.